kit/src/main/java/com/oracle/javafx/scenebuilder/kit/metadata/Metadata.java

```diff
@@ -4470,7 +4472,12 @@ public boolean isPropertyTrimmingNeeded(PropertyName name) {
                 com.oracle.javafx.scenebuilder.kit.metadata.property.value.DoublePropertyMetadata.DoubleKind.COORDINATE,
                 true, /* readWrite */
                 100.0, /* defaultValue */
-                new InspectorPath("Properties", "Specific", 98));
+                new InspectorPath("Properties", "Specific", 10));
+    private final ValuePropertyMetadata useSystemMenuBarNamePropertyMetadata =
+            new BooleanPropertyMetadata(useSystemMenuBarName, 
+                    true, /* readWrite */
+                    false, 
+                    new InspectorPath("Properties", "Specific", 70));
```

Comment:
Change `Specific` to `Node`, value can be 25 or so.

Reply:
Is there a guideline how to assign these properties correctly?

Reply:
Not really. Most of SB features are undocumented, and fixing that is going to require a huge effort...

Reply:
Well, I am learning quite a lot now, will see that I can sketch this and add this to the docs folder within the project.